lib/UserInterface/textinput/TerminalDisplay.h

```diff
@@ -40,6 +40,8 @@ namespace textinput {
     void Detach();
     void DisplayInfo(const std::vector<std::string>& Options);
     bool IsTTY() const { return fIsTTY; }
+    void SetTERM(const char* TERM) { fTERM = TERM; };
+    const char* GetTERM() { return fTERM; };
```

Comment:
These interfaces (nor the data member) should not be needed, see below. Instead we'd need an `IsDumb()`.